polaris/ocean/tests/baroclinic_channel/restart_test/restart_step.py

```diff
@@ -0,0 +1,84 @@
+import time
+
+from polaris.ocean.tests.baroclinic_channel.forward import Forward
+
+
+class RestartStep(Forward):
+    """
+    A forward model step in the restart test case
+    """
+    def __init__(self, test_case, resolution, name):
+        """
+        Create a new test case
+
+        Parameters
+        ----------
+        test_case : polaris.TestCase
+            The test case this step belongs to
+
+        resolution : km
+            The resolution of the test case in km
+
+        name : str
+            the name of the test case
+        """
+        self.resolution = resolution
+        super().__init__(test_case=test_case, name=name, subdir=name, ntasks=4,
+                         min_tasks=4, openmp_threads=1,
+                         resolution=resolution)
+
+    def dynamic_model_config(self, at_setup):
+        """
+        Add model config options, namelist, streams and yaml files using config
+        options or template replacements that need to be set both during step
+        setup and at runtime
+
+        Parameters
+        ----------
+        at_setup : bool
+            Whether this method is being run during setup of the step, as
+            opposed to at runtime
+        """
+        super().dynamic_model_config(at_setup)
```

Comment:
Can you explain to me why this line is included here but not in `forward.py`? It seems that this line is unnecessary here because `ntasks` and `min_tasks` are hard-coded so `update_namelist_pio` wouldn't do anything.